corehq/apps/repeaters/repeater_generators.py

```diff
@@ -315,8 +315,6 @@ def get_test_payload(self, domain):
 
 class UserPayloadGenerator(BasePayloadGenerator):
 
-    deprecated_format_names = ('user_json',)
```

Comment:
@dannyroberts I checked for these on prod and india and didn't find anything.  The `all_docs/by_doc_type` view isn't synced to the receiverwrapper db, unfortunately, so I had to go domain-by-domain.